File: develop-api/src/main/java/com/salesianos/triana/backend/Animangav4/dtos/PasswordDto.java
package com.salesianos.triana.backend.Animangav4.dtos;

import com.salesianos.triana.backend.Animangav4.validation.annotation.PasswordsMatch;
import lombok.*;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@PasswordsMatch(passwordField = "passwordNew",
        verifyPasswordField = "passwordNewVerify",
        message = "{user.password.notmatch}")
public class PasswordDto {

    @NotBlank(message = "{user.password.blank}")
    private String password;
    @Size(min = 8, message = "{user.password.size}")
    @NotBlank(message = "{user.password.blank}")
    private String passwordNew;
    @Size(min = 8, message = "{user.password.size}")
    @NotBlank(message = "{user.password.blank}")
    private String passwordNewVerify;
}

File: develop-mobile/animangav5frontend/lib/models/password_dto.dart
class PasswordDto {
  PasswordDto({
    required this.password,
    required this.passwordNew,
    required this.passwordNewVerify,
  });
  late final String password;
  late final String passwordNew;
  late final String passwordNewVerify;

  Map<String, dynamic> toJson() {
    final _data = <String, dynamic>{};
    _data['password'] = password;
    _data['passwordNew'] = passwordNew;
    _data['passwordNewVerify'] = passwordNewVerify;
    return _data;
  }
}